packs/data/bestiary-effects.db/effect-discerning-aura.json

```diff
@@ -43,7 +40,7 @@
         },
         "target": null,
         "tokenIcon": {
-            "show": true
+            "show": false
```

Comment:
?

Reply:
Since it was already apparent when the effect was applied, I figured the icon was just visual clutter. Works fine either way. :)